Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -455,6 +456,11 @@ - (void)_initializeWithConfiguration:(WKWebViewConfiguration *)configuration
     // FIXME: This is a workaround for <rdar://135515434> to prevent the tint color from being set to either solid black or white.
     self.tintColor = UIColor.systemBlueColor;
 #endif
+
+#if PLATFORM(IOS_FAMILY)
+    if (WTF::IOSApplication::isFeedly())
```

Comment:
Even though it's just one app for now, I wonder if we should abstract this into a method? `needsPointerTouchCompatibility`?

Just seems slightly strange to have the application check in directly in the initializer 😅